packages/origin-backend/src/pods/organization/organization.controller.ts

```diff
@@ -190,6 +190,25 @@ export class OrganizationController {
         };
     }
 
+    @Get('/:id/invitations')
+    @UseGuards(AuthGuard())
+    async getInvitationsForOrganization(
+        @Param('id') organizationId: string
+    ): Promise<IOrganizationInvitation[]> {
+        const organization = await this.organizationService.findOne(organizationId);
```

Comment:
This allows any logged in user to browse every organization invitations. I think this should not be possible. 

You can use UserDecorator and read the organization id of the logged in user.